mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -186,14 +197,17 @@ def build_generic(self, *,
         inst : Any
             A new instance of the `cls` class.
         """
+        tunables_path = config.get("include_tunables")
+        if tunables_path is not None:
+            tunables = self._load_tunables(tunables_path, tunables)
         (class_name, class_config) = self.prepare_class_load(config, global_config)
         inst = instantiate_from_config(base_cls, class_name, tunables, service, class_config)
         _LOG.info("Created: %s", inst)
         return inst
 
     def build_environment(self, config: Dict[str, Any],
+                          tunables: TunableGroups,
                           global_config: Optional[Dict[str, Any]] = None,
-                          tunables: Optional[TunableGroups] = None,
```

Comment:
Made this required argument to support catching places where we weren't building up the TunableGroups along the way.